src/main/java/com/untamedears/JukeAlert/storage/JukeAlertLogger.java

```diff
@@ -1632,11 +1632,19 @@ public boolean updateMutedGroups(UUID uuid, String group2Mute) {
 			return null;
 		} else {
 			// Add the first value
-			ignoringUsers.add(set.getString(1));
+			String[] mutedGroups = set.getString(2).split("\\s+");
+			Set<String> mutedGroupsSet = new HashSet<String>(Arrays.asList(mutedGroups));
```

Comment:
Ye I just wanted the O(1) on .contains(), probably irrelevant as there's only so many groups a player will have muted.

BTW, Arrays.asList() is O(1) and so is the .contains() on HashSet no?